repository-data/webfiles/src/main/resources/site/freemarker/common/scripts/hub-filter/hub-filter-controller.js.ftl

```diff
@@ -0,0 +1,252 @@
+<#ftl output_format="HTML">
+
+<#include "./hub-query-filter.js.ftl"/>
+<#include "./hub-tag-filter.js.ftl"/>
+
+<script>
+    (function() {
```

Comment:
Are there any acceptance test for this new JavaScript?

Reply:
There are acceptance tests for the search and filter functionality, let me check where you find them.

Reply:
I did not ask about unit tests. Good to know QA are adding acceptance tests.

Reply:
@GeoffHayward Acceptance tests cover the search, filter and highlight functionality as per the comment on the Jira ticket -> https://jira.digital.nhs.uk/browse/DW-1163?focusedCommentId=580209&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-580209

Reply:
Same as above.